### PR TITLE
Updated config.yml to include more resources

### PR DESCRIPTION
**Pull request by utkershrajvenshi**
# Pull Request Template

## Description

Updated ISSUE_TEMPLATE/config.yml in FLINT_UI to include more resources of moja global.

## Testing

Navigate to .github/ISSUE_TEMPLATE/ and open config.yml to find updated content.

### CODE REVIEW

**Comment by utkershrajvenshi** [comment]
Please review this PR @chicken-biryani

**Comment by chicken-biryani** [comment]
@utkershrajvenshi thanks.

@HarshCasper can you please have a look?


@utkershrajvenshi Harsh may suggest some changes.... Do them, get it reviewed and do the same for the projects shown under the community website.

**Comment by utkershrajvenshi** [comment]
Sure thing. Till @HarshCasper suggests changes, should I continue to update other repos, @chicken-biryani ?

**Comment by chicken-biryani** [comment]
@utkershrajvenshi I would not advice that.

Let the suggestions come in.

After that you could just copy paste in the other repos.

**Comment by utkershrajvenshi** [comment]
okay..

**Comment by utkershrajvenshi** [comment]
@chicken-biryani @HarshCasper any updates?

**Comment by chicken-biryani** [comment]
Hey, @utkershrajvenshi while you're waiting for a review from Harsh... You could help us implement more things at moja global. We do need your help...
What are you comfortable with? Documentation? Implementing UI? Data science?

Let us know... we'll be happy if you could help us.

PS: We will be creating a lot of new issues for FLINT-UI soon. You could join the #user-interface channel on slack to stay updated.

File: .github/ISSUE_TEMPLATE/config.yml
Comment (HarshCasper) [inline .github/ISSUE_TEMPLATE/config.yml:5]: I don't see how this change helps

File: .github/ISSUE_TEMPLATE/config.yml
Comment (HarshCasper) [inline .github/ISSUE_TEMPLATE/config.yml:3]: Change to `Moja global Slack Community`

File: .github/ISSUE_TEMPLATE/config.yml
Comment (HarshCasper) [inline .github/ISSUE_TEMPLATE/config.yml:6]: `Moja global documentation`

File: .github/ISSUE_TEMPLATE/config.yml
Comment (HarshCasper) [inline .github/ISSUE_TEMPLATE/config.yml:8]: `Technical Documentation for moja global`

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml
Comment (utkershrajvenshi) [reply .github/ISSUE_TEMPLATE/config.yml:5]: I was following a template shared by @chicken-biryani  for updating config.yml
But I get that there's no change in semantic meaning of either one. Should I keep this or change it back to the initial term? @HarshCasper

Comment (chicken-biryani) [reply .github/ISSUE_TEMPLATE/config.yml:5]: Hi, @utkershrajvenshi I believe it can be removed.
Once this PR is done I believe you can do the same across the projects.

**Pull request closed without merging**
